vogue/server/utils.py

```diff
@@ -59,48 +59,66 @@ def get_percentiles(samples: list, key: str)-> float:
 
     return percentiles
 
+def build_app_tag_group_queries()-> dict:
```

Comment:
```suggestion
def build_app_tag_group_queries()-> list:
```
It says below that it returns a list of application tags, not a dict